components/brave_wallet_ui/page/screens/onboarding/components/onboarding-content-layout/onboarding-content-layout.style.tsx

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import * as leo from '@brave/leo/tokens/css'
+import Icon from '@brave/leo/react/icon'
+
+export const StyledWrapper = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  @media (min-width: 768px) {
+    overflow: hidden;
+  }
+`
+
+export const HeaderWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: ${leo.spacing.xl};
+  justify-content: center;
+  align-items: center;
+  padding: 0px 40px 0 40px;
```

Comment:
can this be reduced?
```suggestion
  padding: 0px 40px;
```